grpc-interface-support/src/python.py

```diff
@@ -41,7 +45,49 @@ def get_required_sdk_version_python() -> str:
     return sdk_version
 
 
+def get_template_dir() -> str:
+    return os.path.join(
+        get_package_path(), "grpc-interface-support", "data", "templates", "python"
+    )
+
+
+class GrpcCodeExtractor:
+    """
+    Provides methods for extracting code from generated gRPC python files.
+    """
+
+    def __init__(
+        self, proto_file: ProtoFileHandle, base_path: str, source_path: str = "."
+    ):
+        self.__proto_file = proto_file
+        self.__base_path = base_path
+        self.__source_path = source_path
+
+        self.grpc_source_path = os.path.join(
+            self.__base_path,
+            self.__source_path,
+            f"{self.__proto_file.get_service_name().lower()}_pb2_grpc.py",
+        )
+
+    def create_source_stub_code(self) -> List[str]:
+        service_name = to_camel_case(self.__proto_file.get_service_name())
```

Comment:
This will possibly lead to errors because with the VAL service if the generator works as in c++ it will not find it due to conversion.

Reply:
Removed the to_camel_case